lib/SymbolGraphGen/DeclarationFragmentPrinter.cpp

```diff
@@ -136,7 +136,13 @@ void DeclarationFragmentPrinter::printStructurePre(PrintStructureKind Kind,
 void DeclarationFragmentPrinter::printTypeRef(Type T, const TypeDecl *RefTo,
     Identifier Name,
     PrintNameContext NameContext) {
-  openFragment(FragmentKind::TypeIdentifier);
+  if (Kind != FragmentKind::Attribute) {
+    openFragment(FragmentKind::TypeIdentifier);
+  } else {
+    // create a separate fragment so that only the attribute name is linkable
+    closeFragment();
```

Comment:
Is this actually needed? This was previously outputting both @ and the identifier, so I would have thought all you'd need here is to open the fragment with Attribute instead of TypeIdentifier?

Reply:
`openFragment()` doesn't actually create a new fragment if the type you give it is the same as the current type.

Reply:
Ah I see 👍